src/amo/components/SearchResult/styles.scss

```diff
@@ -104,6 +105,10 @@ $icon-default-size: 32px;
   width: 100%;
   word-wrap: break-word;
 
+  .RecommendedBadge {
+    @include margin-start(10px);
```

Comment:
Did you try `8px` or `12px`? On the frontend we try to stick to a standard set of sizes (there are exceptions in older code).

Reply:
Yeah, sorry, I just threw `10px` in there in a whim. `8px` looks better, thanks.